api/mapping/views.py

```diff
@@ -272,6 +272,8 @@ def get_serializer_class(self):
             # use the edit serialiser when the user tries to alter the scan report
             return ScanReportEditSerializer
         return super().get_serializer_class()
+    filter_backends = [DjangoFilterBackend]
+    filterset_fields = {"parent_dataset": ["exact"]}
```

Comment:
A small stylistic change. Could you put these 2 class variables at the top of the class please? I think it makes more sense to have them there like in the other views.